### PR TITLE
template ids added for aat, demo and ithc envs

### PR DESCRIPTION
**Pull request by btensay**
https://tools.hmcts.net/jira/browse/RET-3168

The changes in this pr point to updated or new templates. Hence, the new/updated templates have additional personalisation field. To resolve correctly, this pr presumes having the code changes in et-sya-api (https://github.com/hmcts/et-sya-api/pull/305) deployed. 

**Does this PR introduce a breaking change?** (check one with "x")

```
[X] Yes
[ ] No
```

### CODE REVIEW

**Comment by hsjhita** [comment]
@btensay I think we should only set the templates in the other environments once the code has been merged in

**Comment by hsjhita** [comment]
@btensay refer to my previous comment. the template can be perftest but should only be in the other environments when merged in otherwise it will fail on those environments

**Review by JustMehmet — DISMISSED** [review]

File: apps/et/et-sya-api/ithc.yaml [hunk 1]
```diff
@@ -11,4 +11,4 @@ spec:
       environment:
         SUBMIT_CASE_EMAIL_TEMPLATE_ID: 10c31d51-c518-4d73-b2c3-1c02771ef41e
         CY_SUBMIT_CASE_EMAIL_TEMPLATE_ID: f61f7f2a-4825-491b-bc48-0299f1a8ba23
-        SUBMIT_CASE_DOC_UPLOAD_ERROR_EMAIL_TEMPLATE_ID: 3007a1e9-13b0-4bf9-9753-398ea91b8564
+        SUBMIT_CASE_DOC_UPLOAD_ERROR_EMAIL_TEMPLATE_ID: 3007a1e9-13b0-4bf9-9753-398ea91b8564
```
Comment (hsjhita) [inline apps/et/et-sya-api/ithc.yaml:14]: also don't need this but can leave in

**Review by hsjhita — APPROVED** [review]

File: apps/et/et-sya-api/ithc.yaml (hunk 1 above)
Comment (btensay) [reply apps/et/et-sya-api/ithc.yaml:14]: I agree. I thought it would not matter.

**Pull request merged**